Dockerfile.launcher

```diff
@@ -0,0 +1,12 @@
+# To build it:
+#  `docker build -t codenvy/che-launcher -f Dockerfile.launcher .`
+# To use it:
+#  `docker run --rm \
+#       -v /var/run/docker.sock:/var/run/docker.sock \
+#        codenvy/che-launcher [start|stop|restart|update]
+FROM docker:1.11
```

Comment:
How would you tag these images with different Docker version? `codenvy/che-launcher:release-docker1.11` ?

My main concerns is that Users should check their Docker version before running Che. We should really provide a simple command that users can copy/paste.

Let me check what Rancher and Docker UCP do because they must have the same problem.